docs/jwst/associations/asn_from_list.rst

```diff
@@ -18,36 +17,38 @@ Command Line
 Usage
 -----
 
-Level2 Associations
-^^^^^^^^^^^^^^^^^^^
+Stage 2 Associations
```

Comment:
There is a note in Overview that says it is called "stage" now and "level" is old doc, so might as well update?